src/netconfig.cpp

```diff
@@ -379,7 +381,7 @@ static const Command ifconfigCommands[] = {
     {"reset", nullptr, "Reset configuration to factory defaults", cmdReset},
     {"mac", "{INTERFACE} MAC", "Set MAC address", cmdMac},
     {"hostname", "NAME", "Set host name", cmdHostname},
-    {"gateway", "IP", "Set default gateway", cmdGateway},
+    {"gateway", "IP", "Set default gateway", cmdGateway, "NOTE: The system doesn't allow unreachable gateways. Please ensure that your \ngateway belongs to the network defined by the IP address and the netmask."},
```

Comment:
@nest1ing @ikmsk10 Done as requested removed cepitalization from netmask and gateway moved into new line

Reply:
I've seen. But in the source this line is still too long.

Reply:
> I've seen. But in the source this line is still too long.

@nest1ing Then I am confused a little, what do you wish me to do?
Do you want me not only move the "gateway"-starting part in the help text output to the new line, but also break the line in source code onto two lines?

Like so:
![BreakLine](https://user-images.githubusercontent.com/61800770/192052824-90ee89e6-2f2b-4e41-bb1c-137a0fe6c437.png)